src/common/image.c

```diff
@@ -1166,21 +1162,6 @@ static uint32_t dt_image_import_internal(const int32_t film_id, const char *file
     g_free(extra_file);
   }
 
-  // insert dummy image entry in database
```

Comment:
Likewise...

Reply:
Same context as above.
But this comment is misleading (the image is not dummy).

Reply:
So in image.c that's just some cleanup without behavior change.
Not directly related to metadata import.
Let me know if want me to revert this.